### PR TITLE
Docs:  explain `.cast` file in README 

### PR DESCRIPTION
**Pull request by Penggeor**

### CODE REVIEW

File: static/cast/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+## Cast file
+
+The file ended with `.cast` come from [Asciinema](https://asciinema.org/) library which records and display terminal sessions.
```
Comment (92hackers) [inline static/cast/README.md:3]: Here, maybe `files` should be used, we will have not only one cast file.

File: static/cast/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+## Cast file
+
+The file ended with `.cast` come from [Asciinema](https://asciinema.org/) library which records and display terminal sessions. 
+
+## Usage
+1. Cast file can be imported as a module:
```
Comment (92hackers) [inline static/cast/README.md:6]: Every  cast file

File: static/cast/README.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+```js
+  import("asciinema-player").then((player) => {
+    player.create(hlnListStacksCast, data, {
+      /** Code */
+    }
+  }
+```
+
+⛳️ More specific detail please refer this component: `/src/components/Asciinema/index.tsx`.
```
Comment (92hackers) [inline static/cast/README.md:23]: detail ->  details

Comment (92hackers) [reply static/cast/README.md:23]: refer  -> refer to

**Review by 92hackers — CHANGES_REQUESTED** [review]

File: static/cast/README.md (hunk 1 above)
Comment (Penggeor) [reply static/cast/README.md:3]: Here is just to explan what is "cast file".

Comment (92hackers) [reply static/cast/README.md:3]: come  -->  comes

display  -->  displays

File: static/cast/README.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
+## Cast file
+
+The file ended with `.cast` comes from [Asciinema](https://asciinema.org/) library which records and displays terminal sessions.
```
Comment (92hackers) [inline static/cast/README.md:3]: The file ended with `.cast` comes from [Asciinema](https://asciinema.org/) library which records and displays terminal sessions.   -->

The `.cast` files are downloaded from  [Asciinema](https://asciinema.org/), they are used by developers to record and replay terminal sessions.

**Review by 92hackers — CHANGES_REQUESTED** [review]

**Pull request merged**